src/utils/common.js

```diff
@@ -3,4 +3,28 @@ const calculateColumnSpan = (imageCount, imagesPerRow) => {
   return totalColumns
 }
 
-export { calculateColumnSpan }
+const convertFilter = (filter) => {
+  if (filter === 'V' || filter === 'gp') {
```

Comment:
This function maps the potential filters into 'r', 'g', 'b'. We can expand this as we accept more filter names to mean those colors.